src/EditorFeatures/Core/ChangeSignature/AbstractChangeSignatureCommandHandler.cs

```diff
@@ -131,14 +139,15 @@ private static void HandleResult(ChangeSignatureResult result, Solution oldSolut
                 // wait context. That means the command system won't attempt to show its own wait dialog 
                 // and also will take it into consideration when measuring command handling duration.
                 context.OperationContext.TakeOwnership();
-                finalSolution = previewService.PreviewChanges(
+                finalSolution = await previewService.PreviewChangesAsync(
                     string.Format(EditorFeaturesResources.Preview_Changes_0, EditorFeaturesResources.Change_Signature),
                     "vs.csharp.refactoring.preview",
                     EditorFeaturesResources.Change_Signature_colon,
-                    result.Name,
+                    result.Name ?? "",
```

Comment:
To other comment: not sure if this went nullable or not.

Reply:
I'll update the interface to nullable enable as well

Reply:
Ah, I got slightly confused. I think I'll just let topLevelName be null in the service and default to blank string in the service